BEProjectsApp/models.py

```diff
@@ -1,6 +1,24 @@
 from django.db import models
 from django.utils import timezone
 
+DOMAIN_OHP = [
```

Comment:
Isn't this and `DOMAIN_CHOICES` the same? As it also covers the same domains.

Reply:
exactly i was going to create the list Nirav didn't have to do it
If you want ill delete mine then

Reply:
Yes, you can delete yours. But good code practice on the list to tuple thing.

Reply:
okay will do